common-app-lib/src/cmr/common_app/services/kms_fetcher.clj

```diff
@@ -11,15 +11,15 @@
                      {...}]}
          :providers [...]}"
   (:require
+   [clojure.string :as string]
    [cmr.common-app.services.kms-lookup :as kms-lookup]
    [cmr.common.cache :as cache]
    [cmr.common.log :refer [error]]
    [cmr.common.redis-log-util :as rl-util]
    [cmr.common.util :as util]
    [cmr.redis-utils.config :as redis-config]
    [cmr.redis-utils.redis-cache :as redis-cache]
-   [cmr.transmit.kms :as kms])
-  (:import (clojure.lang ExceptionInfo)))
```

Comment:
Are you sure you don't need the import for the exception?  Usually these have been put into the namespaces to avoid the class not found exception. when the compiler removes the ExceptionInfo since it thinks that it isn't being used.

Reply:
when is this error message generated, at run time or compile time?

Reply:
Resolving because this was undone but git is not showing as Outdated